kernel_version_unsupported.go

```diff
@@ -0,0 +1,13 @@
+// +build !linux
+
+package ebpf
+
+var ErrNonLinux = fmt.Errorf("unsupported platform %s/%s", runtime.GOOS, runtime.GOARCH)
```

Comment:
You are missing imports in this file:

```suggestion
import (
	"fmt"
	"runtime"
)

var ErrNonLinux = fmt.Errorf("unsupported platform %s/%s", runtime.GOOS, runtime.GOARCH)
```

Reply:
👍